apis/v1alpha2/policy_types.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+// PolicyTargetReference identifies an API object to apply policy to.
```

Comment:
I think we should have some (brief) guidance on how this struct should be used, with a pointer to the GEP, as a start.

I also think that we should actually supply template objects for Policy objects, for Route objects, and any other one that we expect people to do their own versions of. But that's a separate thing I'll log an issue about.

Reply:
> I also think that we should actually supply template objects for Policy objects, for Route objects, and any other one that we expect people to do their own versions of. But that's a separate thing I'll log an issue about.

Heavy +1
As an implementer, I want to simply embed the struct in my struct and call it done.

Reply:
Good call, I've added a more detailed description here + a reference to the docs that already include a sample policy object.